packages/jest-environment/src/index.ts

```diff
@@ -29,7 +29,6 @@ export type ModuleWrapper = (
   require: Module['require'],
   __dirname: string,
   __filename: Module['filename'],
-  global: Global.Global,
```

Comment:
`global` is already defined (https://github.com/facebook/jest/blob/e84a70c8456d61daa36b5c79a4df7ee752d6d6d6/packages/jest-environment-node/src/index.ts#L34), so this isn't breaking

Reply:
heh, as I wrote in an edit in the OP - it breaks for JSDOM as that does _not_ have this code